framework/structures.md

```diff
@@ -118,70 +122,78 @@ The encryption context MUST reserve the following key fields for use by the AWS
 The encryption context SHOULD reserve any key field with the prefix `aws` for use by AWS KMS and
 other AWS services.
 
-### Encryption Materials
+### Data Key Materials
 
 #### Implementations
 
-- [Java](https://github.com/aws/aws-encryption-sdk-java/blob/master/src/main/java/com/amazonaws/encryptionsdk/model/EncryptionMaterials.java)
-- [Python](https://github.com/aws/aws-encryption-sdk-python/blob/master/src/aws_encryption_sdk/materials_managers/__init__.py)
-- [C](https://github.com/aws/aws-encryption-sdk-c/blob/master/include/aws/cryptosdk/materials.h)
-- [Javascript](https://github.com/awslabs/aws-encryption-sdk-javascript/blob/master/modules/material-management/src/cryptographic_material.ts)
+- [Java](https://github.com/aws/aws-encryption-sdk-java/blob/master/src/main/java/com/amazonaws/encryptionsdk/DataKey.java)
 
-#### Structure
+TODO: The above is only a similar data structure related to the deprecated master key/master key provider concepts.
 
-Encryption materials are a structure containing materials needed for [encryption](#encrypt.md).
-This structure MAY include any of the following fields:
+##### Algorithm Suite
```

Comment:
Yeah I agree - I'm thinking more that I would rather express the required relationship between input and output values, rather than duplicating values just so the requirement is only on the output as a separate structure.